lib_test/test_headers.ml

```diff
@@ -40,7 +40,35 @@ let test_remove () =
       "a");
 ;;
 
+let test_ci_equal () =
+  let ascii = List.init (0x7f + 1) Char.chr in
```

Comment:
Do we care about testing all possible characters? I am fairly sure we'll accept any character as being part of the header, but I don't know if most of those are out of spec or what.

Reply:
I expanded the range and swapped the implementation with one that works for values > `0x7f`.